.gitignore

```diff
@@ -6,4 +6,6 @@ libminiocpp.so
 *.a
 *.so
 build/
-*.pc
\ No newline at end of file
+*.pc
+.vs
+CMakeSettings.json
```

Comment:
where is CMakeSettings.json? and why should it be ignored?

Reply:
i use visual studio 2019 (msvc), will generate these directories and files

Reply:
Okay understood.